NAMESPACE

```diff
@@ -2,8 +2,26 @@
 
 export("%>%")
 export(auto_aux_update)
+export(cl_validate_raw)

```

Comment:
Is there a reason to export all the validation functions? I think they are meant to be used internally. If they are, you should use `#' @keywords internal` in the roxygen chunk of those functions. If they are not, please provide a short explanation. Thanks !

Reply:
As you have suggested, I included  #' @keywords internal to roxygen chunk to all the data validation functions.

Thanks